sql/moz-fx-data-shared-prod/mozilla_vpn_derived/waitlist_funnel_v1/query.sql

```diff
@@ -0,0 +1,102 @@
+WITH send_ids AS (
+  SELECT
+    SendID,
```

Comment:
Does a `SendID` identify an email that was sent, or is it unique to the email and recipient?

Reply:
It appears that it's indeed an email that was sent.